BlueprintUILists/Sources/Element+Item.swift

```diff
@@ -0,0 +1,137 @@
+//
+//  Element+Item.swift
+//  BlueprintUILists
+//
+//  Created by Kyle Van Essen on 7/24/22.
+//
+
+import BlueprintUI
+import ListableUI
+
+
+// MARK: Item / ItemContent Extensions
+
+extension Element {
+    
+    /// Converts the given `Element` into a Listable `Item` with the provided ID. You can use this ID
+    /// to scroll to or later access the item through the regular list access APIs.
+    /// You many also optionally configure the item, setting its values such as the `onDisplay` callbacks, etc.
+    ///
+    /// ```swift
+    /// MyElement(...)
+    ///     .item(id: "my-provided-id") { item in
+    ///         item.insertAndRemoveAnimations = .scaleUp
+    ///     }
+    /// ```
+    ///
+    /// ## ⚠️ Performance Considerations
+    /// Unless your `Element` conforms to `Equatable` or `IsEquivalentContent`,
+    /// it will return `false` for `isEquivalent` for each content update, which can dramatically
+    /// hurt performance for longer lists (eg, more than 20 items): it will be re-measured for each content update.
+    ///
+    /// It is encouraged for these longer lists, you ensure your `Element` conforms to one of these protocols.
+    public func item(
```

Comment:
Can I suggest `listItem`, since this will show up for anyone import Listable and `item` is fairly amiguous

```suggestion
    public func listItem(
```

Reply:
Good point, agreed. I'll change the other one to `listHeaderFooter` to be consistent too?

Reply:
Done!